<html>
<head>
  <title>Symposium Programme</title>
  <!-- Printable front cover (in HTML) -->
  <meta charset="UTF-8">
  <meta name="Author" content="Christian Steinruecken">
  <style type="text/css"><!--
  /* base settings */
  body {
    margin:0;
    padding:0;
    font-family:sans-serif;
    background-color: #dcebf7; color: #000000;
    background-color: #91B9A4; color: #000000;
    background-color: #ffffff; color: #666666;
  }
  .page_ind:after { content: '?'; }
  @media screen { .corner { padding: 0.5em; } }
  @media print { .corner { padding: 0.5em; } }
  @media screen and (min-width: 100em) {
    body { font-size: 200%; }
    .page_ind:after { content: '> 100em'; }
  }
  @media screen and (max-width: 100em) {
    body { font-size: 160%; }
    .page_ind:after { content: '< 100em'; }
  }
  @media screen and (max-width: 90em) {
    body { font-size: 150%; }
    .page_ind:after { content: '< 90em'; }
  }
  @media screen and (max-width: 80em) {
    body { font-size: 130%; }
    .page_ind:after { content: '< 80em'; }
  }
  @media screen and (max-width: 70em) {
    body { font-size: 120%; }
    .page_ind:after { content: '< 70em'; }
  }
  @media screen and (max-width: 60em) {
    body { font-size: 110%; }
    .page_ind:after { content: '< 60em'; }
  }
  /* PAGE SIZE SPECIFIC SETTINGS */

  /* ISO 216 PAGE SIZES [mm]
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
    4A0     1682 × 2378     –            –         –       –
    2A0     1189 × 1682     –            –         –       –
    A0       841 × 1189     B0      1000 × 1414    C0      917 × 1297
    A1       594 × 841      B1       707 × 1000    C1      648 × 917
    A2       420 × 594      B2       500 × 707     C2      458 × 648
    A3       297 × 420      B3       353 × 500     C3      324 × 458
    A4       210 × 297      B4       250 × 353     C4      229 × 324
    A5       148 × 210      B5       176 × 250     C5      162 × 229
    A6       105 × 148      B6       125 × 176     C6      114 × 162
    A7        74 × 105      B7        88 × 125     C7       81 × 114
    A8        52 × 74       B8        62 × 88      C8       57 × 81
    A9        37 × 52       B9        44 × 62      C9       40 × 57
    A10       26 × 37       B10       31 × 44      C10      28 × 40
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
  */
  @media print and (max-width: 2377mm) and (max-height: 1680mm) { /* 2A0 L */
    body { font-size: 566%; }
    .page_ind:after { content: '2A0 L'; }
  }
  @media print and (max-width: 1680mm) and (max-height: 2377mm) { /* 2A0 P */
    body { font-size: 566%; }
    .page_ind:after { content: '2A0 P'; }
  }
  @media print and (max-width: 1680mm) and (max-height: 1188mm) { /* A0 L */
    body { font-size: 400%; }
    .page_ind:after { content: 'A0 L'; }
  }
  @media print and (max-width: 1188mm) and (max-height: 1680mm) { /* A0 P */
    body { font-size: 400%; }
    .page_ind:after { content: 'A0 P'; }
  }
  @media print and (max-width: 1188mm) and (max-height: 840mm) { /* A1 L */
    body { font-size: 283%; }
    .page_ind:after { content: 'A1 L'; }
  }
  @media print and (max-width: 840mm) and (max-height: 1188mm) { /* A1 P */
    body { font-size: 283%; }
    .page_ind:after { content: 'A1 P'; }
  }
  @media print and (max-width: 840mm) and (max-height: 593mm) { /* A2 L */
    body { font-size: 200%; }
    .page_ind:after { content: 'A2 L'; }
  }
  @media print and (max-width: 593mm) and (max-height: 840mm) { /* A2 P */
    body { font-size: 200%; }
    .page_ind:after { content: 'A2 P'; }
  }
  @media print and (max-width: 593mm) and (max-height: 419mm) { /* A3 L */
    body { font-size: 141%; }
    .page_ind:after { content: 'A3 L'; }
  }
  @media print and (max-width: 419mm) and (max-height: 593mm) { /* A3 P */
    body { font-size: 141%; }
    .page_ind:after { content: 'A3 P'; }
  }
  @media print and (max-width: 419mm) and (max-height: 296mm) { /* A4 L */
    body { font-size: 100%; }
    .page_ind:after { content: 'A4 L'; }
  }
  @media print and (max-width: 296mm) and (max-height: 419mm) { /* A4 P */
    body { font-size: 100%; }
    .page_ind:after { content: 'A4 P'; }
  }
  @media print and (max-width: 296mm) and (max-height: 209mm) { /* A5 L */
    body { font-size: 71%; }
    .page_ind:after { content: 'A5 L'; }
  }
  @media print and (max-width: 209mm) and (max-height: 296mm) { /* A5 P */
    body { font-size: 71%; }
    .page_ind:after { content: 'A5 P'; }
  }
  @media print and (max-width: 209mm) and (max-height: 147mm) { /* A6 L */
    body { font-size: 50%; }
    .page_ind:after { content: 'A6 L'; }
  }
  @media print and (max-width: 147mm) and (max-height: 209mm) { /* A6 P */
    body { font-size: 50%; }
    .page_ind:after { content: 'A6 P'; }
  }
  @media print and (min-width: 2378mm) and (min-height: 1681mm) { /* >2A0 L */
    body { font-size: 800%; }
    .page_ind:after { content: 'huge'; }
  }
  @media print and (min-width: 1681mm) and (min-height: 2378mm) { /* >2A0 P */
    body { font-size: 800%; }
    .page_ind:after { content: 'huge'; }
  }
  @media (orientation: landscape)  {
    .orient_ind:after { content: ' landscape'; }
    .columns { -webkit-column-count: 3; -webkit-column-gap: 0.25em;
               -webkit-column-rule: 0.2em dotted #a0c9eb; /* rgba(0,0,128,0.15); */
               -moz-column-count: 3; -moz-column-gap: 0.5em;
               -moz-column-rule: 0.2em dotted #a0c9eb; /* rgba(0,0,128,0.35); */
               }
  }
  @media (orientation: portrait)  {
    .orient_ind:after { content: ' portrait'; }
    .columns { -webkit-column-count: 2; -webkit-column-gap: 0.25em;
               -webkit-column-rule: 0.2em dotted rgba(0,0,128,0.15);
               -moz-column-count: 2; -moz-column-gap: 0.25em;
               -moz-column-rule: 0.2em dotted rgba(0,0,128,0.35);
               }
  }
  @media print {
    cover {
      display:block;
      padding:0; margin:0; border:0;
      page-break-inside:avoid;
      page-break-before:auto;
      page-break-after:always;
    }
  }
  .black {
    background-color:#000000; color:#ffffff;
  }
  .segment {
    margin:0;
    padding:0.5em;
    padding-left:7.45em;
    padding-right:7.45em;
    padding-right:4.5em;
  }
  .segment h1, .segment h2 {
    margin:0; padding:0;
  }
  .segment.nopad { padding:0 !important; }
  h1 { font-size:320%; }
  h2 { font-size:180%; padding-top:0.75em !important; }
  dim50 { opacity:0.5; }
  dim75 { opacity:0.75; }
  .dg {
    background-color:#106470; color:#ffffff;
    background-color:#117a72; color:#ffffff;
  }
  .mg {
    background-color:#18a69a; color:#ffffff;
  }
  .lg {
    background-color:#1894a6; color:#ffffff;
    background-color:#18a69a; color:#ffffff;
    background-color:#5db3ab; color:#ffffff;
  }
  .bottom {
    position:absolute;
    left:0; right:0; bottom:0;
  }
  --></style>
</head>
<body><cover><div class="segment dg nopad" style="padding-top:5%;">
  <img src="uc-rgb-wtr-4725x982.png"
       style="height:4em; margin-left:3.40937em; margin-top:3.40937em">
  <div class="vspace" style="height:2.5%; clear:both;"> </div>
  <!-- University logo:
       Width: 4725 pixels, height: 982 pixels.
       Text starts at x=992 (20.99%)
       Width of crest: 837 pixels (17.7%), height: 970 pixels.
       Space above logo (to top edge of cover) should be
          837/982 * 4em = 3.40937em.
       Space left of logo (to left edge of cover) should be the same.
       All other text on the page should be aligned to
       (992+837)/982 * 4em = 7.4501em from the left margin.
  -->
</div>
<div class="segment lg" style="padding-top:20%;">
  <h1>Information, Inference, and Energy</h1>
  <h2><dim50>A symposium to celebrate the work of</dim50><br>
      <dim75>Professor Sir David MacKay FRS</dim75></h2>
  <div class="vspace" style="height:10%; clear:both;"> </div>
</div>
<div class="segment mg" style="">
</div>
<div class="segment dg bottom" style="height:10%;">
</div>
</cover>
<div class="content">
  <div class="section" style="">
    <!-- ----------------------------------------------------- -->
    <div class="newpage" style="margin-left:10%;">
      <div class="vspace" style="height:5em;"></div>
      <h2 style="">Programme</h2>
      <h3>Monday 14th</h3>
      <table class="schedule" border="0" cellspacing="0" style="color:inherit; border-collapse:collapse; font-size:inherit;">
      <tr><td class="st">09:15</td><td class="et">09:30</td><td class="ds">Welcome &amp; introduction</td></tr>
      <tr><td class="st">09:30</td><td class="et">10:00</td><td class="ds"><a href="#spiegelhalter">David Spiegelhalter</a></td></tr>
      <tr><td class="st">10:10</td><td class="et">10:50</td><td class="ds"><a href="#counsell">Tom Counsell</a></td></tr>
      <tr><td class="st">10:50</td><td class="et">11:20</td><td class="ds"><i>Coffee &amp; Tea</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">11:20</td><td class="et">12:00</td><td class="ds"><a href="#shokrollahi">Amin Shokrollahi</a></td></tr>
      <tr><td class="st">12:00</td><td class="et">12:40</td><td class="ds"><a href="#sayir">Jossy Sayir</a></td></tr>
      <tr><td class="st">12:40</td><td class="et">12:50</td><td class="ds"><i>Brief reminiscences I</i></td></tr>
      <tr><td class="st">13:00</td><td class="et">14:00</td><td class="ds"><i>Lunch</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">14:00</td><td class="et">14:40</td><td class="ds"><a href="#lynas">Mark Lynas</a></td></tr>
      <tr><td class="st">14:40</td><td class="et">15:20</td><td class="ds"><a href="#frey">Brendan Frey</a></td></tr>
      <tr><td class="st">15:20</td><td class="et">15:30</td><td class="ds"><i>Brief reminiscences II</i></td></tr>
      <tr><td class="st">15:30</td><td class="et">16:00</td><td class="ds"><i>Coffee &amp; Tea</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">16:00</td><td class="et">16:40</td><td class="ds"><a href="#stegle">Oliver Stegle</a></td></tr>
      <tr><td class="st">16:40</td><td class="et">17:20</td><td class="ds"><a href="#durbin">Richard Durbin</a></td></tr>
      <tr><td class="st">17:20</td><td class="et none"></td><td class="ds"><i>(end of Monday programme)</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">19:15</td><td class="et">19:55</td><td class="ds"><i>Pre-dinner drinks (Trinity College)</i></td></tr>
      <tr><td class="st">20:00</td><td class="et">22:30</td><td class="ds"><i>Dinner (Trinity College)</i></td></tr>
      </table>
      <br><br>
    </div>
    <!-- ----------------------------------------------------- -->
    <div class="newpage" style="margin-left:10%;">
      <div class="vspace" style="height:5em;"></div>
      <h2 style="padding-top:5em;">&nbsp;</h2>
      <h3>Tuesday 15th</h3>
      <table class="schedule" border="0" cellspacing="0" style="color:inherit; border-collapse:collapse; font-size:inherit;">
      <tr class="vs"></tr>
      <tr><td class="st">09:30</td><td class="et">10:10</td><td class="ds"><a href="#hopfield">John Hopfield</a></td></tr>
      <tr><td class="st">10:10</td><td class="et">10:50</td><td class="ds"><a href="#brody">Carlos Brody</a></td></tr>
      <tr><td class="st">10:50</td><td class="et">11:20</td><td class="ds"><i>Coffee</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">11:20</td><td class="et">12:00</td><td class="ds"><a href="#herz">Andreas Herz</a></td></tr>
      <tr><td class="st">12:00</td><td class="et">12:40</td><td class="ds"><a href="#kristensson">Per Ola Kristensson</a></td></tr>
      <tr><td class="st">12:40</td><td class="et">12:50</td><td class="ds"><i>Brief reminiscences III</i></td></tr>
      <tr><td class="st">13:00</td><td class="et">14:00</td><td class="ds"><i>Lunch</i></td></tr>
      <tr class="vs"></tr>
      <tr><td class="st">14:00</td><td class="et">14:40</td><td class="ds"><a href="#winn">John Winn</a></td></tr>
      <tr><td class="st">14:40</td><td class="et">15:20</td><td class="ds"><a href="#murray">Iain Murray</a></td></tr>
      <tr><td class="st">15:20</td><td class="et">16:00</td><td class="ds"><a href="#hinton">Geoff Hinton</a></td></tr>
      <tr><td class="st">16:00</td><td class="et">16:30</td><td class="ds">Surprise talk &amp; closing remarks</td></tr>
      <tr class="vs"></tr>
      <!-- <tr><td class="st">16:30</td><td class="et">16:45</td><td class="ds"><i>Group photo</i></td></tr> -->
      <tr><td class="st">16:30</td><td class="et">17:30</td><td class="ds"><i>Coffee and cake reception</i></td></tr>
      <tr><td class="st">17:30</td><td class="et none"></td><td class="ds"><i>Departure</i></td></tr>
      </table>
    </div>
    <!-- ----------------------------------------------------- -->
    <style type="text/css"><!--
        table.schedule a[href] { color:inherit; text-decoration:none; }
        table tr, table td { font-size:inherit; }
        td.st { padding:0 0.5em 0 1em; }
        td.et { padding:0 1em 0 0; opacity:1.0; }
        td.et:before { content: '–'; text-align:center; margin-right:0.5em; }
        td.et.none:before { content: ''; margin-right:0.5em; }
        td.ds { padding:0 1em; }
        tr       { opacity:1.0; height:1.75em; vertical-align:top; }
        tr td.st, tr td.et, tr td.ds { padding-top:0.25em; padding-bottom:0.25em; }
        tr.vs { height: 0.75em !important; }
        tr:hover { background-color:rgba(93,179,171,0.4); }
        tr:hover a[href] { color:rgba(17,122,114,1.0); }
        @media (max-width:20em) {
          td.et { display:none; }
        }
        @media print {
          div.newpage { page-break-after:always; page-break-inside:avoid; }
        }
    --></style>

  </div>


  <div class="section" style="padding-top:5em;">
    <!-- ----------------------------------------------------- -->
    <style type="text/css"><!--
    div.content {
      page-break-inside:auto;
      page-break-before:auto;
      page-break-after:always;
      font-family:serif;
      font-size:125%;
      margin-left:5em;
      margin-right:5em;
      background-color:white;
      color:black;
    }
    div.talk {
      page-break-inside:avoid;
      page-break-before:auto;
      page-break-after:auto;
      text-align:justify;
      margin:-3.5em 0 0 0;
      padding:3.5em 0.5em 2em 0.5em;
      /* border-top:2px solid rgba(127,127,127,0.25); */
    }
    a:nth-child(2n+0) div.talk {
      /* background-color:rgba(223,223,223,0.05); */
    }
    a:nth-child(2n+1) div.talk {
      /* background-color:rgba(32,32,32,0.05); */
      /* color:blue; */
    }
    div.title {
      font-size:110%; font-weight:600;
      margin-top:0.5em;
      padding:1.5em 2.5% 0.35em 0;
    }
    div.speaker {
      font-weight:500;
      font-style:italic;
    }
    div.abstract {
      margin:1em 0.5em 1em 0.5em;
      font-size:95%;
      line-height:140%;
      padding:0 2.5% 0 2.5%;
    }
    --></style>
    <!-- ----------------------------------------------------- -->
    <h2 style="margin-top:3em;">Abstracts</h2>
    <!-- ----------------------------------------------------- -->
    <a name="spiegelhalter"></a><div class="talk">
      <div class="title">Communicating risk using whole numbers</div>
      <div class="speaker">David Spiegelhalter</div>
      <div class="abstract">
        David Mackay's “Sustainable Energy – without the hot air” showed
        the power of translating complex quantities into whole
        numbers that can be added and easily compared.  Similar
        ideas have been tried in communicating risk, the units being
        the Micromort for acute risks and the Microlife for chronic
        risks.  I shall show how these can be used to see how
        recklessly dangerous, or how boringly safe, your life is.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="counsell"></a><div class="talk">
      <div class="title">The Global Calculator: Trying to apply 'Sustainable Energy - without the hot air' to the world</div>
      <div class="speaker">Tom Counsell</div>
      <div class="abstract">
        Could 9 billion people have a rich world standard and style
        of living and yet avoid the worst effects of climate change,
        if only we built enough nuclear power stations? or covered
        enough desert with solar panels? or all used low energy
        light bulbs? or efficient cars? Inspired by David
        Mackay’s 'Sustainable Energy - without the hot air’ a team
        managed by Sophie Hartfield at the UK Department of Energy
        and Climate Change tried to build an interactive calculator
        to help people to figure out the answer.  This talk will
        cover their journey and the results of their attempt.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="shokrollahi"></a><div class="talk">
      <div class="title">Chordal Codes</div>
      <div class="speaker">Amin Shokrollahi</div>
      <div class="abstract">
        Modern electronic devices consist of a multitude of IC
        components: the processor, the memory, the RF modem and the
        baseband chip (in wireless devices), and the graphics
        processor, are only some examples of components scattered
        throughout a device. The increase of the volume of digital
        data that needs to be accessed and processed by such devices
        calls for ever faster communication between these IC's.
        Faster communication, however, often translates to higher
        susceptibility to various types of noise, and inevitably to
        a higher power consumption in order to combat the noise.
        This increase in power consumption is, for the most part,
        far from linear. In this talk I will give a short overview
        of problems encountered in chip-to-chip communication, and
        will  advocate the  use  of  a novel class of codes, called
        “Chordal Codes”, to solve those problems.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="sayir"></a><div class="talk">
      <div class="title">Codes for efficient data storage on DNA molecules</div>
      <div class="speaker">Jossy Sayir</div>
      <div class="abstract">
        Over the past 6 months I have been working with David MacKay
        and with Nick Goldman (EBI) on developing a coding system
        for storing data efficiently on DNA molecules. In this talk,
        I will focus on David’s initial approach to the problem,
        which provides an information theoretic background for the
        achievable storage density and reliability. The assumption
        for this model is a pure packet loss model with noiseless
        recovery of packets, which David called the “fountain
        channel”. This supposes that lower level coding can fully
        compensate for synthesis, sequencing and amplification
        errors such as insertions, deletions and substitutions.  An
        overview of the literature on coding for channels with
        insertions, deletions and substitutions reveals surprisingly
        few publications, the majority of which co-authored by David
        MacKay, and none of which providing unbounded reliability in
        the information theoretic sense. Hence, I will discuss a
        model for DNA storage that combines the packet loss with the
        noisy nature of the recovered packets, and present coding
        strategies for exploiting this channel that David, Nick and
        I have been developing. 
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="lynas"></a><div class="talk">
      <div class="title">Mind the Gap (between science and society) – the case studies of climate, nuclear and GMOs</div>
      <div class="speaker">Mark Lynas</div>
      <div class="abstract">
        Scientists often have a very different idea of risk and
        benefit from policymakers and the general public. With
        special attention to three case studies – climate change,
        nuclear power and genetically modified crops – Mark Lynas
        looks at the gaps between what scientists understand and
        what everyone else thinks, and how these gaps can lead to
        sub-optimal policy outcomes in the real world. 
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="frey"></a><div class="talk">
      <div class="title">Why Medicine Needs Deep Learning</div>
      <div class="speaker">Brendan Frey</div>
      <div class="abstract">
        My research on deep inference and learning reaches back to
        the wake-sleep algorithm, published in 1995, and the paper
        that David MacKay and I wrote in 1996 showing that belief
        propagation in graphs with cycles can be used for accurate
        inference. Most of my time is now spent on genomic medicine.
        Deep learning will transform medicine, but not in the way
        that many advocates think.  The amount of data times the
        mutation frequency divided by the biological complexity and
        the number of hidden variables is small, so downloading a
        hundred thousand genomes and training a neural network won't
        cut it. There is what I call a "genotype-phenotype gap" and
        the value of closing this gap exceeds Google's $200B ad
        market. I believe that the only way to bridge this gap is to
        build machine learning systems that properly incorporate
        biological knowledge. I'll describe this approach, which is
        being pursued by dozens of young investigators, has improved
        our ability to “read the genome”, and will, I believe, be an
        indispensable component in the future of medicine.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="stegle"></a><div class="talk">
      <div class="title">New statistical approaches to disentangle single-cell diversity</div>
      <div class="speaker">Oliver Stegle</div>
      <div class="abstract">
        Many key biological processes are driven by differences in
        the regulatory landscape between single cells. Recent
        technical developments have enabled the transcriptomes and
        epigenomes of hundreds of cells to be assayed in an unbiased
        manner, opening up the possibility to identify and study new
        physiologically relevant, sub-populations of cells. In this
        talk I will discuss statistical advances to elucidate the
        factors that drive single-cell heterogeneity. Accurate and
        scalable latent variable models allow dissecting single-cell
        transcriptome and epigenome studies, thereby disentangling
        biological variation from technical and confounding factors.
        I will illustrate these approaches in applications to large
        datasets consisting of tens of thousands of cells and discuss
        their relationship to methods from population genetics. 
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="durbin"></a><div class="talk">
      <div class="title">New statistical methods for the analysis of genome variation data</div>
      <div class="speaker">Richard Durbin</div>
      <div class="abstract">
        The last two decades have seen an exponential growth in the
        quantity of DNA sequencing data, at a rate more than twice
        as fast as Moore’s law for growth in computing power.
         Individual sequences at one position in the genome are
        related by descent from a common ancestor in a tree, but
        this genealogy is made much more complex across the whole
        genome by recombination, which means that you inherit
        different sections of your genome from different recent
        ancestors.  A clean mathematical model exists for the full
        genealogy but it is too complex to use for inference with
        data sets available today.  I will discuss various
        algorithmic and representational innovations introduced to
        model and exploit genetic relatedness in analysing ever
        larger amounts of genome data, illustrating them with our
        work on the Haplotype Reference Consortium, which has
        assembled over 65,000 whole genome sequences with data at
        nearly 40 million variable sites.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <!-- ----------------------------------------------------- -->
    <!-- ----------------------------------------------------- -->
    <a name="hopfield"></a><div class="talk">
      <div class="title">Emergence, dynamics, and behaviour</div>
      <div class="speaker">John Hopfield</div>
      <div class="abstract">
        How do the principles of psychology relate to the activities
        of 1011 nerve cells in the human brain?  Large physical
        systems generally have emergent collective behaviours.  Real
        computers—the brain included—compute by following a change
        of computer ‘state’ with time.  When that change of state
        can be described by collective variables, robust dynamics
        emerge whose mathematical description can be entirely unlike
        that of the underlying microscopic variables. We illustrate
        the useful emergent computational dynamics of two elementary
        neural networks.  The first deals with the problem of
        variable cadence when recognizing dynamical sensory patterns
        spread over time.  The second achieves goal-directed
        behaviour when multiple candidate goals are simultaneously
        present.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="brody"></a><div class="talk">
      <div class="title">Neural substrates of decision-making in rats</div>
      <div class="speaker">Carlos Brody</div>
      <div class="abstract">
        The most common behavioral observation in decision-making,
        experienced both in our daily lives and in laboratory
        settings, is that easy decisions (where we are likely to
        choose the correct response) are done quickly, whereas
        difficult decisions (less likely to choose correctly) are
        much slower. An appealingly simple model was proposed in the
        behavioral literature many decades ago to account for this
        observation. This model, sometimes known as the “gradual
        accumulation of evidence” model, has been used to explain
        many behavioral data sets. Does the brain implement
        something well approximated by this model? If so, how does
        the brain’s network of neurons actually carry out the
        implementation? We have been using using a combination of
        computational and experimental approaches with rats to try
        to answer these questions.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="herz"></a><div class="talk">
      <div class="title">Decoding the population activity of grid cells for spatial localization and goal-directed navigation</div>
      <div class="speaker">Andreas Herz</div>
      <div class="abstract">
        Mammalian grid cells discharge when an animal crosses the
        points of an imaginary hexagonal grid tessellating the
        environment. I will show how animals can navigate by reading
        out a population vector of such activity patterns across
        multiple spatial scales. The theory explains key
        experimental results about grid cells, makes testable
        predictions for future physiological and behavioural
        experiments, and provides a mathematical foundation for the
        concept of a "neural metric" for space. For goal-directed
        navigation, the proposed allocentric grid cell
        representation can be readily transformed into the
        egocentric goal coordinates needed for planning movements.<br>
        <span style="opacity:0.5;">
        Joint work with Martin Stemmler and Alexander Mathis.
        <br></span>
        <!-- href="http://advances.sciencemag.org/content/1/11/e1500816">http://advances.sciencemag.org/content/1/11/e1500816</a -->
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="kristensson"></a><div class="talk">
      <div class="title">Next-generation text entry</div>
      <div class="speaker">Per Ola Kristensson</div>
      <div class="abstract">
        Text entry is a common everyday computing task. However,
        despite its ubiquitousness it is difficult to devise an
        efficient text entry method that users are willing to adopt.
        In this talk I will explain the narrow design space of text
        entry research and make the case that successful
        next-generation text entry methods are likely to be based on
        designs that merge behavioural solution principles with
        information engineering techniques. I will exemplify this
        principle with several new text entry methods we have
        developed for a variety of use-cases.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="winn"></a><div class="talk">
      <div class="title">Democratising data science</div>
      <div class="speaker">John Winn</div>
      <div class="abstract">
        There has never been a higher demand for data science and
        data scientists.  In science, in medicine, in business, in
        sport, in every area of life, we now have the tools to
        accumulate large quantities of data, but we lack the
        corresponding tools to process and understand it.  I will
        talk about work that we have done in Microsoft Research to
        try to build such tools, the challenges that we have
        encountered and suggest how data science might be made much
        more broadly accessible in the future.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="murray"></a><div class="talk">
      <div class="title">Pseudo-Marginal Slice Sampling</div>
      <div class="speaker">Iain Murray</div>
      <div class="abstract">
        This talk is about sampling from probability distributions
        in a challenging setting: we can only compute noisy (but
        unbiased) estimates of the probability density function at
        individual settings of variables we choose. Previous work on
        such "pseudo-marginal" methods have been used in inference
        problems in genetics, continuous time stochastic processes,
        hierarchical models, and "doubly-intractable" distributions.
        I'll present algorithms that are easier to apply than
        previous work and sometimes work a lot better.<br>
        <span style="opacity:0.5;">
        Joint work with Matt Graham.
        </span>
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <a name="hinton"></a><div class="talk">
      <div class="title">Can sensory cortex do backpropagation?</div>
      <div class="speaker">Geoff Hinton</div>
      <div class="abstract">
        Stochastic gradient descent in multilayer networks of
        neuron-like units has led to dramatic recent progress in a
        variety of difficult AI problems. Now that we know how
        effective backpropagation can be in large networks it is
        worth reconsidering the widely held belief that the cortex
        could not possibly be doing backpropagation. Drawing on
        joint work with Timothy Lillicrap, I will go through the
        main objections of neuroscientists and show that none of
        them are hard to overcome if we commit to representing error
        derivatives as temporal derivatives. This allows the same
        axon to carry information about the presence of some feature
        in the input and information about the derivative of the
        cost function with respect to the input to that neuron. It
        predicts spike-time dependent plasticity and it also
        explains why we cannot code velocity by the rate of change
        of features that represent position.
      </div>
    </div>
    <!-- ----------------------------------------------------- -->
    <!-- ----------------------------------------------------- -->
  </div>
    <!-- ----------------------------------------------------- -->
  <div class="newpage">
    <div class="vspace" style="height:8em;"></div>
    <h2>Sponsors</h2>
    This symposium was generously supported by:
    <style><!-- a[href] { color:inherit; text-decoration:none; } --></style>
    <ul style="list-style-type:none; line-height:1.55em; margin-left:4%;">
      <li><a href="http://mlg.eng.cam.ac.uk/">The Machine Learning Group</a> (CBL, CUED)</li>
      <li><a href="http://www.eng.cam.ac.uk/research/academic-divisions/information-engineering">The Information Engineering Division</a> (CUED)</li>
      <li><a href="http://www.eng.cam.ac.uk/">The Engineering Department</a></li>
      <li><a href="http://www.cl.cam.ac.uk/">The Computer Laboratory</a></li>
      <li><a href="http://www.trin.cam.ac.uk/">Trinity College</a></li>
      <li><a href="http://www.phy.cam.ac.uk/">The Cavendish Laboratory</a></li>
      <li><a href="http://research.microsoft.com/en-us/labs/cambridge/">Microsoft Research Cambridge</a></li>
    </ul>
    <div class="vspace" style="height:5em;"></div>
    <h2>Organisers</h2>
    <ul style="list-style-type:none; line-height:1.55em; margin-left:4%;">
      <li>Zoubin Ghahramani</li>
      <li>Oliver Stegle</li>
      <li>Alan Blackwell</li>
      <li>Christian Steinruecken</li>
    </ul>
  </div>
</div></body>
</html>
